objects/obj_pause/Draw_77.gml

```diff
@@ -0,0 +1,47 @@
+//Disable alpha blending (Drawing just RGB)
+gpu_set_blendenable(false);
+
+if (pause_is_paused)  //draw frozen image to screen while paused

```

Comment:
I would put this check after the below check as it will only ever be run after the pause button has been pressed.